draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -772,8 +760,13 @@ free to choose any congestion control algorithm as long as they follow the
 guidelines specified in {{Section 3 of ?RFC8085}}, and QUIC implementors make
 use of this freedom.
 
-For real-time traffic, it is best that the QUIC implementation use a congestion controller that
-seeks to minimize queueing delays.
+For real-time traffic, it is best that the QUIC implementation use a congestion
+controller that aims at keeping queues, and thus the latency, at intermediary
+network elements as short as possible. Delay-based congestion control algorithms
+use, for example, an increasing one-way delay as a signal of impending
```

Comment:
I'd suggest "might use, for example", because I don't know that's true for all of them, but otherwise, this looks lovely. 

One minor point - you said "hidden in the terminology section", and IIRC, I started adding subsections as part of the Introduction, before I realized that (in total) the terminology section would appear after NINE subsections. Does it make sense to you to move Section 1.2 ("Motivations") into its own top-level section, following the terminology section?

Reply:
Thanks, I updated the PR.

Yes, we could do that. But let's make a separate PR for that. I can prepare it.